### DISCUSSION TITLE
Azure OpenAI not working

### DISCUSSION THREAD

**Discussion by ipaidfairly** [lobehub/lobehub#2174] [Self Hosting]
Hi there

i'm trying to add a Azure OpenAI model to lobechat

But it keeps giving an error  `{
  "error": {
    "code": "DeploymentNotFound",
    "message": "The API deployment for this resource does not exist. If you created the deployment within the last 5 minutes, please wait a moment and try again.",
    "deployId": "gpt-3.5-turbo"
  },
  "endpoint": "https://openai-sweden-kc.openai.azure.com/",
  "provider": "azure"
}`

i've attached my settings below, any ideas ? 

![image](https://github.com/lobehub/lobe-chat/assets/124269471/570cd785-f32b-40d2-b1ad-dbb35b17c60a)
![image](https://github.com/lobehub/lobe-chat/assets/124269471/21f3c56a-08a4-4b35-b5fa-f46211eef7b7)

[a comment by an automated account is not shown]

**Comment by ConnorCHN** [comment]
Azure openai has no your model on your screenshot, you may change it to azure openai model, plz check your deploy. It look like "gpt-4" or "gpt-35-0125". It could never be called by "gpt-lobe".

**Comment by ipaidfairly** [comment]
I see, I thought it was the name of the model deployment in Azure, I'll try your suggestion
